dallinger/db.py

```diff
@@ -69,3 +70,33 @@ def init_db(drop_all=False):
     Base.metadata.create_all(bind=engine)
 
     return session
+
+
+def serialized(func):
+    """Run a function within a db transaction using SERIALIZABLE isolation.
```

Comment:
@davisagli It seems that `session.connection()`'s behaviour is undefined when a transaction is already in progress, in that the DB chooses between ROLLBACK or COMMIT. Do you think we should be explicit about this decorator being for routes only, to avoid people calling functions that have been decorated from other code?